src/Controls/src/Core/MultiBinding.cs

```diff
@@ -18,6 +18,7 @@ public sealed class MultiBinding : BindingBase
 		BindableObject _proxyObject;
 		BindableProperty[] _bpProxies;
 		bool _applying;
+		private bool _updates;
```

Comment:
Can we give this variable a more descriptive name? We check if it is `false` and return early in some cases, but it's not clear why.

Reply:
Agreed. I'll get that fixed shortly. Just an oversight.

Reply:
Changed. Let me know if that's not going to work for you.